recipes/librdmacm-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: librdmacm-cos6-x86_64
+  version: 1.0.21
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/librdmacm-1.0.21-0.el6.x86_64.rpm
+    sha256: 39887f875e4e34964d842a519805de08c39f198862f3dc4f81bb9b804660fa34
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/librdmacm-1.0.21-0.el6.src.rpm
+    folder: source
+
+build:
+  number: 0
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+
+###########################################################
+# Commented out to avoid a conda-build bug.               #
+# xref: https://github.com/conda/conda-build/issues/3553  #
+###########################################################
+#
+#test:
+#  commands:
+#    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
```

Comment:
Currently adding a `test` section results in issue ( https://github.com/conda/conda-build/issues/3553 ). So it is commented for now.

Reply:
Have since added a workaround to add the tests back.